### ISSUE TITLE
Doesn't act like "Alt-Tab switcher" as suggested on help page

### ISSUE THREAD

**Issue by boamaod** [soulthreads/xwinmosaic#28]
Currently help page says that with command line option `-R` or `--persistent` you can make xwinmosaic act as an Alt-Tab switcher:

> Make XWinMosaic acts like Alt-Tab switcher

However if you try to bind it to Alt-Tab in practise (tried Openbox keybindings and xbindkeys), there are multiple problems and in the end I find the description of this option rather confusing:

* If you press Alt-Tab, you cannot continue moving between windows back and forth without releasing Alt key like with Alt-Tab switchers, but have to release Alt and Tab to actually start navigation.
* It might be almost acceptable, if without releasing Alt I were able to use arrow keys to navigate, but even that doesn't work.
* Releasing Tab, keeping Alt down and then pressing Tab again closes xwinmosaic if used in combination with keybinding tools referred above.

I suggest rephrasing the description of the Alt-Tab option and say it just daemonizes the application and nothing more, or much better, implement the true Alt-Tab switcher behaviour and add it as a separate option (or default behaviour in case of daemonizing).

My Openbox configuration experiment:
```
    <!-- Keybindings for window switching -->
    <keybind key="A-Tab">
      <action name="If">
        <query>
          <name>xwinmosaic</name>
        </query>
        <else>
          <action name="execute">
            <command>xwinmosaic</command>
            <action name="Focus"/>
            <action name="Raise"/>
            <action name="Unshade"/>
          </action>
        </else>
      </action>
    </keybind>
```

Or maybe you can share some example keybinding options for generic GNU/Linux desktop environments to make it work as an Alt-Tab switcher? Am I missing something here?

**Comment by soulthreads** [comment]
Honestly, this feature was only fully implemented and tested for Windows by the interested party, so I'm not really surprised that it doesn't work in X11.
But your point is valid, and I'll see if there is a way to make it work properly.

**Comment by ArtifTh** [comment]
Yes, this feature was tested only on Windows, where this software installs system-wide hook at Alt-Tab keypress by itself and runs constantly in background (there also is small memory leak somewhere). This feature for X11 was never implemented.-- Best wishes, Timur Sultanov23.12.2016, 18:05, "Anton Lobashev" <notifications@github.com>:Honestly, this feature was only fully implemented and tested for Windows by the interested party, so I'm not really surprised that it doesn't work in X11.
But your point is valid, and I'll see if there is a way to make it work properly.

**Comment by boamaod** [comment]
If somebody comes up with changes that may fix it for X11, I'd be happy to test them. At the moment I don't have time to dig into it myself, but I was also thinking of implementing some UX features like lousy arrows (if you press up and there is no box directly above currently highlighted one, it still moves diagonally up etc). Making the locations/areas of different applications persistent with some fuzzy grouping would be interesting feature too, but this can be implemented in script level as far as I understand the logic of the application.
